### ISSUE TITLE
Docstrings go after class: or def: lines

### ISSUE THREAD

**Issue by remram44** [madeleineudell/D3M-DimensionalityReduction#2]
I understand that we don't all use Python as our primary language, however this is the correct syntax:
```
class DimReducer:
    """
    This class implements a trivial dimensionality reduction method
    """
```

**Comment by madeleineudell** [comment]
Thanks, I appreciate the reminder :)

**Comment by madeleineudell** [comment]
Closed by 85c0d117fdf14a02ad8870029c3a8a90694805e6
